### ISSUE TITLE
Plugin not available in repository?

### ISSUE THREAD

**Issue by brnl** [JeffyW/OctoPrint-ServerStats#1]
Hi @JeffyW,

I'm trying to find this plugin in the plugin manager, but it doesn't shows up. Is it actually available?

Thanks
brnl

**Comment by JeffyW** [comment]
Sorry it took so long to get back to you.  I took a hiatus from printing / SoC programming.

I haven't published it yet because I didn't consider it "done" yet.
I only had a Pine A64 to test on at the time, but now have a PiZeroW and Pi3B+ to do some testing on to make sure everything looks reasonable.  Using these I've found a bunch of bugs.
I also want to clean up the APIs a bit, pass more information (such as a friendly name) and make sure that any consumer can depend on the data being of the same format regardless of board.  In my branch of NavBarTemp, I want no logic that knows about hardware type.

I started back up recently and have a bunch of fixes I've been testing.  I've verified them using the "Install new Plugins... ... from URL" in OctoPrint, using the "Download as zip" link from GitHub.  I'll make sure to update the main page and merge the branches so everything works.
